test/sicmutils/expression/render_test.cljc

```diff
@@ -102,7 +102,7 @@
   (is (= "x^y" (s->infix (expt 'x 'y)))))
 
 (deftest more-with-D
-  (af/with-literal-functions [f g [p [] 0]]
+  (af/with-literal-functions [f g]
```

Comment:
I found this during a head-scratching moment trying to decode the syntax for the three arg version (it's name, domain, range of fn), and realized... I don't think an abstract function of NO arity can do anything...

whatever the answer is `p` is not used in the body here, so we can delete.